app/coreAPI.server.ts

```diff
@@ -298,3 +298,53 @@ export const oneClick = async (
     throw e;
   }
 };
+
+export type BrandDto = {
+  uuid: string;
+  receiverName: string;
+  logoImageUrl: string;
+  homepageUrl: string;
+  primaryColor: string;
+};
+
+/**
+ * Get a brand DTO by uuid.
+ * @param brandUuid Brand uuid.
+ * @param accessToken Access token to access core service API.
+ * @returns
+ */
+const mapBrandDto = (brandDto: any): BrandDto => ({
```

Comment:
Why is brandDto of type `any`? Isn't the api returning a `BrandDto` already?

Reply:
Having core-types package to it would solve this.